src/hooks/index.js

```diff
@@ -0,0 +1,2 @@
+export * from './use-force-update'
```

Comment:
👍 -- was going to ask if you thought that these would be worth exposing externally, but my hunch is probably not ;)